lib/debug/deprecate.js

```diff
@@ -34,6 +35,14 @@ const assert = require('./assert');
  * - `url` A URL that refers to additional information about the deprecation
  */
 function deprecate(description, condition, options) {
+  const isDeprecationRemoved = semver.gt(emberCLIVersion, options.until);
```

Comment:
I think this should be `gte` -- the until is the version at which it breaks

Reply:
I also think this needs to check the `options.for` is `ember-cli` -- this method is public and can be used by addons